src/objects/Customizable_Rollup_Settings__c.object

```diff
@@ -199,6 +199,45 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Rollups_Contact_HardCredit_ChunkSize__c</fullName>
+        <description>Only used by the new Customizable Rollups engine in LDV Orgs and only when required for LDV orgs</description>
```

Comment:
I would recommend giving a bit more detail as to what this is for. Can we combine the description and help text and just use the same value for both. "LDV ORGS ONLY: If chunking is needed to avoid a Query Timeout in the batch job, specifies the number of Contacts processed in each batch. Used by the Customizable Rollups engine."

Reply:
Thanks @smarovich. How about the following?
* Field Name: `LDV: Contact Hard Credit Chunk Size`
* API Name: `ChunkSize_Contact_HardCredit__c`
* Inline Help Text: `FOR LDV ORGS ONLY WITH SFDO GUIDANCE. To avoid a Query Timeout in the rollup batch job in orgs with many millions of Contacts, this specifies the maximum number of Contacts processed in each batch.`